src/core/components/ChatView.tsx

```diff
@@ -224,7 +224,7 @@ export const ChatView = ({
               </div>
             </div>
 
-            <div className={styles.importantInfo} data-testid="importantInfo">
```

Comment:
Will do a follow-on sometime this week to see how many more of these can be removed.